src/audio/module_adapter/module/volume/volume_hifi3.c

```diff
@@ -10,20 +10,20 @@
  * \authors Tomasz Lauda <tomasz.lauda@linux.intel.com>
  */
 
-#include <sof/audio/volume.h>
-
-#if defined(__XCC__) && XCHAL_HAVE_HIFI3
-
 #include <sof/audio/buffer.h>
 #include <sof/audio/component.h>
-#include <sof/audio/volume.h>
 #include <sof/common.h>
 #include <ipc/stream.h>
-#include <xtensa/tie/xt_hifi3.h>
 #include <stddef.h>
 #include <stdint.h>
 
-LOG_MODULE_DECLARE(volume, CONFIG_SOF_LOG_LEVEL);
+LOG_MODULE_DECLARE(volume_hifi3, CONFIG_SOF_LOG_LEVEL);
+
+#include <sof/audio/volume.h>
+
+#if defined(__XCC__) && XCHAL_HAVE_HIFI3
```

Comment:
'#if' ... '#endif' ?

Reply:
Its not a new block, #ifdef just shifted a few lines, #endif untouched